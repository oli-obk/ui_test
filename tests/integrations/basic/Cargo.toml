[package]
name = "basic"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
# A proc-macro dependency, needs special treatment.
serde_derive = "1.0"
# `quote` now shows up both as host and target dependency -- let's make sure that works.
quote = "1.0.25"

[dev-dependencies]
ui_test = { path = "../../.."}
tempfile = "3.3.0"

[[test]]
name = "ui_tests"
harness = false

[[test]]
name = "json"
harness = false

[[test]]
name = "run_file"
harness = true

# Regression check for #225
[lib]
crate-type = ["staticlib", "lib"]
