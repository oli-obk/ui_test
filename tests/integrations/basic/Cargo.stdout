
running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

{ "type": "suite", "event": "started" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/aux_attr_proc_macro.rs () - tests/actual_tests/aux_attr_proc_macro.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/aux_derive.rs () - tests/actual_tests/aux_derive.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/aux_proc_macro.rs () - tests/actual_tests/aux_proc_macro.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/dep_derive.rs () - tests/actual_tests/dep_derive.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/error_above.rs () - tests/actual_tests/error_above.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/error_below_multi_rev.rs () - tests/actual_tests/error_below_multi_rev.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/executable.rs () - tests/actual_tests/executable.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/foomp-rustfix.rs () - tests/actual_tests/foomp-rustfix.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/foomp.rs () - tests/actual_tests/foomp.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/joined_above.rs () - tests/actual_tests/joined_above.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/joined_below.rs () - tests/actual_tests/joined_below.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/joined_mixed.rs () - tests/actual_tests/joined_mixed.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/mac_span.rs () - tests/actual_tests/mac_span.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/match_diagnostic_code.rs () - tests/actual_tests/match_diagnostic_code.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/no_rustfix.rs () - tests/actual_tests/no_rustfix.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/rustfix-multiple.rs () - tests/actual_tests/rustfix-multiple.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/stdin.rs () - tests/actual_tests/stdin.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/unicode.rs () - tests/actual_tests/unicode.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/windows_paths.rs () - tests/actual_tests/windows_paths.rs" }
{ "type": "test", "event": "started", "name": "tests/actual_tests/subdir/aux_proc_macro.rs () - tests/actual_tests/subdir/aux_proc_macro.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/aux_attr_proc_macro.rs (Building aux file tests/actual_tests/auxiliary/proc_macro_attr.rs) - tests/actual_tests/aux_attr_proc_macro.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/aux_attr_proc_macro.rs () - tests/actual_tests/aux_attr_proc_macro.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/aux_derive.rs (Building aux file tests/actual_tests/auxiliary/derive_proc_macro.rs) - tests/actual_tests/aux_derive.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/aux_derive.rs () - tests/actual_tests/aux_derive.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/aux_derive.rs () - tests/actual_tests/aux_derive.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/aux_proc_macro.rs (Building aux file tests/actual_tests/auxiliary/the_proc_macro.rs) - tests/actual_tests/aux_proc_macro.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/aux_proc_macro.rs () - tests/actual_tests/aux_proc_macro.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/dep_derive.rs () - tests/actual_tests/dep_derive.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/dep_derive.rs (run) - tests/actual_tests/dep_derive.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/error_above.rs () - tests/actual_tests/error_above.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/error_below_multi_rev.rs () - tests/actual_tests/error_below_multi_rev.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/error_below_multi_rev.rs (edition2015) - tests/actual_tests/error_below_multi_rev.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/error_below_multi_rev.rs (edition2018) - tests/actual_tests/error_below_multi_rev.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/executable.rs () - tests/actual_tests/executable.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/executable.rs (run) - tests/actual_tests/executable.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/foomp-rustfix.rs () - tests/actual_tests/foomp-rustfix.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/foomp-rustfix.rs () - tests/actual_tests/foomp-rustfix.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/foomp.rs () - tests/actual_tests/foomp.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/joined_above.rs () - tests/actual_tests/joined_above.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/joined_above.rs () - tests/actual_tests/joined_above.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/joined_below.rs () - tests/actual_tests/joined_below.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/joined_below.rs () - tests/actual_tests/joined_below.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/joined_mixed.rs () - tests/actual_tests/joined_mixed.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/joined_mixed.rs () - tests/actual_tests/joined_mixed.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/mac_span.rs () - tests/actual_tests/mac_span.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/mac_span.rs () - tests/actual_tests/mac_span.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/match_diagnostic_code.rs () - tests/actual_tests/match_diagnostic_code.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/match_diagnostic_code.rs () - tests/actual_tests/match_diagnostic_code.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/no_rustfix.rs () - tests/actual_tests/no_rustfix.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/rustfix-multiple.rs () - tests/actual_tests/rustfix-multiple.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/rustfix-multiple.rs () - tests/actual_tests/rustfix-multiple.1.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/rustfix-multiple.rs () - tests/actual_tests/rustfix-multiple.2.fixed" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/stdin.rs () - tests/actual_tests/stdin.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/stdin.rs (run) - tests/actual_tests/stdin.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/unicode.rs () - tests/actual_tests/unicode.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/windows_paths.rs () - tests/actual_tests/windows_paths.rs" }
{ "type": "test", "event": "ok", "name": "tests/actual_tests/subdir/aux_proc_macro.rs () - tests/actual_tests/subdir/aux_proc_macro.rs" }
{ "type": "suite", "event": "ok", "passed": 33, "failed": 0, "ignored": 0, "measured": 0, "filtered_out": 0 }

running 3 tests
...
test result: ok. 3 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

Building aux file tests/actual_tests/auxiliary/proc_macro_attr.rs ... ok
tests/actual_tests/aux_attr_proc_macro.rs ... ok
Building aux file tests/actual_tests/auxiliary/derive_proc_macro.rs ... ok
tests/actual_tests/aux_derive.rs ... ok
tests/actual_tests/aux_derive.fixed.rs ... ok
Building aux file tests/actual_tests/auxiliary/the_proc_macro.rs ... ok
tests/actual_tests/aux_proc_macro.rs ... ok
tests/actual_tests/dep_derive.rs ... ok
tests/actual_tests/dep_derive.rs (revision `run`) ... ok
tests/actual_tests/error_above.rs ... ok
tests/actual_tests/error_below_multi_rev.rs ... ok
tests/actual_tests/error_below_multi_rev.rs (revision `edition2015`) ... ok
tests/actual_tests/error_below_multi_rev.rs (revision `edition2018`) ... ok
tests/actual_tests/executable.rs ... ok
tests/actual_tests/executable.rs (revision `run`) ... ok
tests/actual_tests/foomp-rustfix.rs ... ok
tests/actual_tests/foomp-rustfix.fixed.rs ... ok
tests/actual_tests/foomp.rs ... ok
tests/actual_tests/joined_above.rs ... ok
tests/actual_tests/joined_above.fixed.rs ... ok
tests/actual_tests/joined_below.rs ... ok
tests/actual_tests/joined_below.fixed.rs ... ok
tests/actual_tests/joined_mixed.rs ... ok
tests/actual_tests/joined_mixed.fixed.rs ... ok
tests/actual_tests/mac_span.rs ... ok
tests/actual_tests/mac_span.fixed.rs ... ok
tests/actual_tests/match_diagnostic_code.rs ... ok
tests/actual_tests/match_diagnostic_code.fixed.rs ... ok
tests/actual_tests/no_rustfix.rs ... ok
tests/actual_tests/rustfix-multiple.rs ... ok
tests/actual_tests/rustfix-multiple.1.fixed.rs ... ok
tests/actual_tests/rustfix-multiple.2.fixed.rs ... ok
tests/actual_tests/stdin.rs ... ok
tests/actual_tests/stdin.rs (revision `run`) ... ok
tests/actual_tests/unicode.rs ... ok
tests/actual_tests/windows_paths.rs ... ok
tests/actual_tests/subdir/aux_proc_macro.rs ... ok

test result: ok. 33 passed


running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

